diagnostic.md

```diff
@@ -73,13 +89,20 @@ data in JSON format.
 ```
 
 ```sh
-# your answer here
+curl --include --request PUT http://www.example.com/donuts/  'ID'='42' 'NAME'='French Cruller' 'PRICE'='$0.99'
```

Comment:
"POST" is the action. the way it would look would be 
```sh
curl --include --request "POST" "http://example.com/donuts" \
  --header "Content-Type: application/json" \
  --data $'{
    "donut": {
      "name": "French Cruller",
      "price": "$0.99"
    }
  }'
```